style.css

```diff
@@ -829,4 +890,672 @@
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    grid-area: come2;
+  }
+
+  .about-container .social-media {
+    display: flex;
+    flex-direction: column;
+    grid-area: come3;
+  }
+
+  .about-container .social-media .social-text {
+    width: 120px;
+    height: 24px;
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .about-container .social-media .media-icons {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    width: 200px;
+    height: 40px;
+    justify-content: space-between;
+  }
+
+  .about-myself .about-container .get-myResume {
+    margin-top: 24px;
+    grid-area: come4;
+  }
+
+  .about-myself .about-container .get-myResume a {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    border: 1px solid #396df2;
+    border-radius: 16px;
+    width: 138px;
+    height: 24px;
+    padding: 16px;
+    text-decoration: none;
+  }
+
+  .about-myself .about-container .language {
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+    width: 544px;
+    height: 72px;
+  }
+
+  .top-skill {
+    grid-area: come5;
+  }
+
+  .about-myself .about-container .language-list {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    text-decoration: none;
+    gap: 12px;
+    margin-left: -190px;
+  }
+
+  .about-myself .about-container .language-list .list {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    justify-content: center;
+    background: #f7f7f9;
+    border-radius: 8px;
+    width: 122px;
+    height: 120px;
+    padding: 16px;
+    list-style: none;
+  }
+
+  footer .footer-container p {
+    font-family: 'Poppins', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #ebebff;
+    text-align: center;
+    width: 569px;
+    height: 102px;
+  }
+
+  .about-myself .about-container .language p {
+    /* width: 237px; */
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #253858;
+    width: 98px;
+    height: 24px;
+  }
+
+  .about-myself .about-container .language-list .list img {
+    width: 48px;
+    height: 48px;
+  }
+
+  .about-myself .about-container .Frameworks,
+  .skills {
+    width: 327px;
+    height: 72px;
+    border-bottom: 1px solid #dfe1e6;
+  }
+
+  .about-myself .about-container .Frameworks {
+    grid-area: come7;
+    width: 544px;
+  }
+
+  .about-myself .about-container .skills {
+    grid-area: come8;
+  }
+
+  /* footer menu */
+
+  footer {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    width: 100vw;
+  }
```

Comment:
Kindly center the submit button to match the figma design.
This is how it looks now.
![image](https://user-images.githubusercontent.com/33268927/172818390-356527cc-6e6a-427b-85b4-0d3be26b9d1d.png)